### DISCUSSION TITLE
Setting a Track/Playlist to play at a certain Time of Day

### DISCUSSION THREAD

**Discussion by itsbhanusharma** [AzuraCast/AzuraCast#3676] [Q&A]
Hi Awesome People,

We're currently trying out azuracast for an online community radio project. I have a lot of experience with Sourcefabric Airtime which allowed us to schedule a certain Item to be played at a given time of day in a very intuitive calendar page. 

I tried searching through various issues here but couldn't find anything concrete. 

Basically, what I'm looking for is the ability to play a certain Audio Recording at a certain Date/Time. we do not want it to repeat or replay at any time, Just want it to automatically go on air at its scheduled time and then basically never repeat unless explicitly scheduled to be played again. 

I'm not an expert with liquidsoap but my understanding is that airtime was based on liquidsoap as well so there may be some way to make it happen with Azuracast as well, most likely it is an omission at my part that I'm unable to find the exact instructions to achieve the desired result. 

Any help or pointers will be highly appreciated :)

and Kudos to the developers for making such an awesome piece of software.

**Comment by nathanfriend** [comment]
Hi itsbhanusharma,
We’re also a community station that’s used Airtime and trying out AzureCast.  Create a playlist then edit it.  Click the schedule tab at the top, then click the Add scheduled item button.  Select when you want it to playout. On the advanced tab check the interrupt to start on time checkbox. Then associate music files with the playlist.

Cheers,

Nathan.
